<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color:rgb(177,198,200);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 75 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>WELCOME, </string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
font: 75 20pt &quot;Microsoft JhengHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reserveSlotButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(144, 191, 200); font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Reserve Slot</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>312</width>
     <height>183</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCalendarWidget QToolButton {

height: 20px;
width: 100px;
color: white;
font-size: 15px;
icon-size: 36px, 36px;
background-color: blue;
}

QCalendarWidget QWidget{

alternate-background-color: rgb(255, 255, 255);

}

QCalendarWidget QAbstractItemView:enabled {

font-size: 12px;
color: rgb(0, 0, 0);
background-color: white;
selection-background-color: rgb(85, 170, 255);
selection-color:rgb(255, 85, 0);
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;Microsoft YaHei Light&quot;;</string>
   </property>
   <property name="text">
    <string>Doctor Available:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DoctorNameComboX">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findDoctorButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(144, 191, 200); font-size:14pt;</string>
   </property>
   <property name="text">
    <string>Find Doctors</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
